imsv-docs-astro/src/content/docs/guides/authentication.mdoc

```diff
@@ -54,220 +55,276 @@ authenticated by supplying the access token in the `Authorization` header.
-    are presented to the user when signing the challenge message and are used to
-    determine the level of access the user is granting to the application. The
-    following scopes are available:
+#### 2. Login
+For returning users, the client application initiates
+the login process by sending a request to the [Initiate Login API](/api-reference/initiate-login). The response contains a
+challenge message to be signed by the user's wallet. Users must first
+authenticate with this endpoint before performing other actions such as creating
+a funding source.
 
-    - `cardholder-partner` - Manage cards within your {partnerName} Immersve account
+When sending a login or sign up request, authorization scopes must be passed. These scopes
+are presented to the user when signing the challenge message and are used to
+determine the level of access the user is granting to the application. The
+following scopes are available:
 
-    For example:
+- `cardholder-partner` - Manage cards within your {partnerName} Immersve account
 
+For example:
+
+{% tab key="auth-method" %}
+  {% item title="Siwe" %}
+    {% code title="Challenge message" %}
+      ```
+      app.immersve.com wants you to sign in with your Ethereum account:
+      0xA3058369d6A481B1ff08F62B352409c3D709De9b
+
+      Sign in with Ethereum to the app. This request will not trigger a blockchain transaction or cost any gas fees.
+
+      URI: https://app.immersve.com
+      Version: 1
+      Chain ID: 1
+      Nonce: 2hFm7TDbZmerUgnrJ
+      Issued At: 2022-08-11T22:29:48.244Z
+      ```
+    {% /code %}
+  {% /item %}
+  {% item title="Algorand" %}
+    {% code title="Challenge message" %}
+      ```
+      GlSDzZHcxrSDeJbKu: Login with your algorand address 0xA3058369d6A481B1ff08F62B352409c3D709De9b
+      ```
+    {% /code %}
+  {% /item %}
+  {% item title="Xrpl" %}
     {% code title="Challenge message" %}
-    ```
-    app.immersve.com wants you to sign in with your Ethereum account:
-    0xA3058369d6A481B1ff08F62B352409c3D709De9b
-
-    Sign in with Ethereum to the app. This request will not trigger a blockchain transaction or cost any gas fees.
-
-    URI: https://app.immersve.com
-    Version: 1
-    Chain ID: 1
-    Nonce: 2hFm7TDbZmerUgnrJ
-    Issued At: 2022-08-11T22:29:48.244Z
-    ```
+      ```
+      //encoded transaction
+      7321ED86C55E77C731F33313A3A47A7E...
+      ```
     {% /code %}
+  {% /item %}
+{% /tab %}
 
-    #### 2. Sign Message
-    Invoke the wallet's message signing capability to get a signature for the given challenge message. The specific nature
-    of the invocation of the message signing function is specific to the particular wallet in use.
+#### 3. Sign Message
+Invoke the wallet's message signing capability to get a signature for the given challenge message.
+The specific nature of the invocation of the message signing function is specific to the particular wallet in use.
```

Comment:
```suggestion
Each network does message signing differently.
```
I think this sentence could be removed entirely, or changed to more simply point out that each network does message signing differently.